spinalcordtoolbox/compat/launcher.py

```diff
@@ -34,4 +35,4 @@ def main():
             mpi_flags = "-n 1"
         cmd = ["mpiexec"] + mpi_flags.split() + cmd
 
-    os.execvpe(cmd[0], cmd[0:], env)
+    return subprocess.call(" ".join(cmd), shell=True, env=env)
```

Comment:
From discussions in https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/22#discussion_r811346511, I learned that `shell=True` is perhaps not so good.

I'm not sure why I have this here, but I should remember to test with `shell=False` instead.